src/components/navbar/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { NavbarStyled, NavItemStyled } from '../../styled/NavbarStyled';
+// import NavItemStyled from './NavItemStyled';
+import { Link } from 'react-router-dom'
+
+const NavBar = (props) => {
+  return (
+    <NavbarStyled>
+      <NavItemStyled>
+        <Link to='/'>
```

Comment:
NavLink could help you here, it lets you assign an activeClassName that will assign that class when route match, and has the same Link's functionality